nixops/__main__.py

```diff
@@ -527,6 +527,16 @@
     help="include the physical specification in the evaluation",
 )
 
+subparser = add_subparser(
+    subparsers, "eval", help="eval the given file is nix code in the network expression"
```

Comment:
```suggestion
    subparsers, "eval", help="evaluate a Nix expression with the NixOps network as arguments"
```

my proposed wording isn't quite right, but maybe something like this?